utilities/cloudharness_utilities/deployment-configuration/helm/templates/auto-services.yaml

```diff
@@ -3,26 +3,25 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .app.name | quote }}
+  name: {{ .app.harness.service.name | quote }}
   labels:
-    app: {{ .app.name | quote }}
+    app: {{ .app.harness.deployment.name | quote }}
 {{ include "deploy_utils.labels" .root | indent 4 }}
 spec:
   selector:
-    app: {{ .app.name | quote }}
+    app: {{ .app.deploymentname | quote }}
   ports:
-    - port: {{ .app.port }}
+    - port: {{ .app.harness.deployment.port }}
       name: http
 {{- end }}
 
 
 {{- range $app := .Values.apps }}
-  {{- if and (hasKey $app "port") ($app.autoservice | default true)  }}
+  {{- if and (hasKey $app "port") $app.autoservice  }}
```

Comment:
is this correct? port is in the harness part of app

Reply:
You're right. This is wrong but still working because old values are kept

Reply:
I think we should also add autoservice to the template with a default false

Reply:
Addressed